packages/jsii-dotnet-runtime/build.sh

```diff
@@ -4,7 +4,7 @@ set -euo pipefail
 # embed jsii-runtime as a resource
 bundle_dir="src/Amazon.JSII.Runtime/jsii-runtime"
 mkdir -p ${bundle_dir}
-rsync -av node_modules/jsii-runtime/webpack/ ${bundle_dir}
+rsync -av $(node -p 'require("path").dirname(require.resolve("jsii-runtime/package.json"))')/webpack/ ${bundle_dir}
```

Comment:
Don't understand why this doesn't have to be `-pe` ?

Reply:
`-p` behaves like `-e`, so in fact it is not necessary to add the `-e`.